hackathon_site/dashboard/frontend/src/components/general/AlertBox/AlertBox.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import AlertBox from "components/general/AlertBox/AlertBox";
+import { render } from "testing/utils";
+
+const mockErrorList = [
```

Comment:
Are these errors supposed to mock what a user might potentially have as reason for faulty hardware?

Reply:
No, these are just random strings, they're used for making sure they display properly in the AlertBox component, take a look at the tests to see how they're used.